components/linear/actions/get-teams/get-teams.mjs

```diff
@@ -0,0 +1,19 @@
+import linearApp from "../../linear.app.mjs";
+
+export default {
+  key: "linear-get-teams",
+  name: "Get Teams",
+  description: "Get all the teams",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    linearApp,
+  },
+  async run({ $ }) {
+    const { nodes: teams } = await this.linearApp.listTeams();
+
+    $.export("summary", `Found ${teams.length} teams`);
```

Comment:
```suggestion
    $.export("summary", `Found ${teams.length} team(s)`);
```
Just in case you found only one record. Or maybe you can add an `if` checking the length to use `team` or `teams`. Your call :)